FoodFite/Models/UserProfile.cs

```diff
@@ -13,11 +13,21 @@ namespace FoodFite.Models
     public class UserProfile
     {
         public string Name { get; set; }
+        public string Opponent { get; set; }
+        public string Weapon { get; set; }
         public List<Item> Inventory { get; set;}
         public double Health { get; set;}
         public Protection Clothes { get; set; }
+        public Dictionary<string, Food> FoodMap { get; set; }
```

Comment:
What is the use of this and FoodInventory? Inventory and ListFood() should cover these

Reply:
It's a look up for when a food is selected. All of our inputs are text, so if I select a grapes, I need a way to look up my grapes object. This also allows merging of foods for the future.

Reply:
FoodMap should probably live elsewhere but I guess this works for now